### PR TITLE
Feature/cosmetic

### PR DESCRIPTION
**Pull request by martinfalk**
various cosmetic changes like updating workspaces and visualizers to the latest module versions.
* modified volume raycasting visualizer
* updated example workspaces (`stream_tubes` now based on `Integral Lines to Mesh` processor)

Stream Tubes:

Hurricane Isabel:

### CODE REVIEW

File: modules/opengl/src/debugmessages.cpp [hunk 1]
```diff
@@ -192,9 +192,11 @@ void handleOpenGLDebugMode(Canvas::ContextID context) {
     auto mode = openglSettings->debugMessages_.getSelectedValue();
     auto severity = openglSettings->debugSeverity_.getSelectedValue();
     setOpenGLDebugMode(mode, severity);
-    logDebugMode(mode, severity, context);
+    if (mode != debug::Mode::Off) {
+        logDebugMode(mode, severity, context);
```
Comment (ResearchDaniel) [inline modules/opengl/src/debugmessages.cpp:196]: Should not logDebugMode handle off mode?

**Review by ResearchDaniel — APPROVED** [review]
One minor comment

File: modules/opengl/src/debugmessages.cpp (hunk 1 above)
Comment (martinfalk) [reply modules/opengl/src/debugmessages.cpp:196]: This is for the initial message when a canvas is created. The reasoning here is that you don't want to see this message every time a canvas is created (applies to canvas processor, port inspectors, etc.)

Comment (martinfalk) [reply modules/opengl/src/debugmessages.cpp:196]: On the other hand, you might want to see the status change from "debug" to "off".

**Pull request merged**